### PR TITLE
CI pipeline with travis

### PR DESCRIPTION
**Pull request by pichardoJ**
## Changes

- `test` script changed to `test:watch` in order to have a single run test script
- Added `.travis.yml` file with basic npm deployment configuration
  - Will only deploy tagged commits on master branch

## Before Merge

- [ ] Change NPM credentials

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+languaje: node_js
```
Comment (ilyaztsv) [inline .travis.yml]: I think that adding caching node_modules will be great :)

File: package.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
     "min-js": "google-closure-compiler-js dist/radi.js > dist/radi.min.js",
     "min-es": "squash dist/radi.es.js -o dist/radi.es.min.js",
     "lint": "eslint ./src/**/*.js --fix",
-    "test": "jest --watch",
+    "test:watch": "jest --watch",
```
Comment (ilyaztsv) [inline package.json]: I assume that combine `npm run build` and `npm run test` is good choice for CI. For example,  `"ci": "npm run test:ci && npm run build"` where  `"test:ci": "npm run test -- --ci"`

Please see [ci](https://facebook.github.io/jest/docs/en/cli.html#ci)

**Comment by pichardoJ** [comment]
Yeah I agree, will update today

**Comment by pichardoJ** [comment]
Should we also update `.npmignore` to not include docs?

**Comment by Marcisbee** [comment]
Closing in favor of #27

**Pull request closed without merging**